server/highlight.py

```diff
@@ -79,3 +100,18 @@ def format_range_unified(start, stop):
                 yield from insert(j1, j2)
             elif tag == 'equal':
                 yield from equal(i1, i2, j1, j2)
+
+def diff_files(files_before, files_after, diff_type):
```

Comment:
This can be cached, keyed on (assignment_id, backup_id, diff_type). I removed the cache on the frontend because the comments may change.